### PR TITLE
Improved compatibility of fonts and colors

### PR DESCRIPTION
**Pull request by engelhamer**
## Description
- Replaced the font imports with rel stylesheets for better cross-browser compatibility (especially targeted at Safari and Firefox)
- Replaced the light-grey theme color with the Bootstrap default to make the overall interface more consistent

---
This change is [Reviewable](https://reviewable.io/reviews/ejourn-al/ejournal/1270)

### CODE REVIEW

File: src/django/templates/email_templates/head.html [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
-    <title></title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">    <title></title>
```
Comment (Larspolo) [inline src/django/templates/email_templates/head.html:6]: Think the title should be on the next row

File: src/django/templates/email_templates/base.html [hunk 2]
```diff
@@ -3,8 +3,9 @@
     <head>
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <link href="https://fonts.googleapis.com/css?family=Roboto|Overpass" rel="stylesheet">
-        <title></title>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">        <title></title>
```
Comment (Larspolo) [inline src/django/templates/email_templates/base.html:8]: Same issue here as in the other PR. Title seems on the wrong line

**Review by Larspolo — APPROVED** [review]
Categories display leaked, but otherwise looks good

File: src/vue/src/components/category/CategoryDisplay.vue [hunk 3]
```diff
@@ -6,43 +6,20 @@
...
-                @click.native="$emit('select-category', category)"
-                @remove-category="$emit('remove-category', category)"
-                @show-info="
-                    infoCategory = $event
-                    $nextTick(() => { $bvModal.show(infoModalID) })
-                "
-            />
-        </div>
-        <slot/>
+        <category-tag
```
Comment (mjvkeulen) [inline src/vue/src/components/category/CategoryDisplay.vue:9]: What's changed? (not in description or screenshot)

File: src/django/templates/email_templates/base.html [hunk 4]
```diff
@@ -3,8 +3,9 @@
     <head>
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <link href="https://fonts.googleapis.com/css?family=Roboto|Overpass" rel="stylesheet">
-        <title></title>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
[the remark this answers is not in the document]
Comment (engelhamer) [reply src/django/templates/email_templates/base.html:7]: This snippet is generated by Google Fonts.

[the remark this answers is not in the document]
Comment (engelhamer) [reply src/django/templates/email_templates/base.html:7]: This snippet is generated by Google Fonts.

File: src/vue/src/components/category/CategoryDisplay.vue (hunk 3 above)
Comment (engelhamer) [reply src/vue/src/components/category/CategoryDisplay.vue:9]: Yeah this is the category preview, accidentally branched from there.

File: src/django/templates/email_templates/base.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (mjvkeulen) [reply src/django/templates/email_templates/base.html:7]: So is it configured correctly for our use?

[the remark this answers is not in the document]
Comment (engelhamer) [reply src/django/templates/email_templates/base.html:7]: Yes, this is the standard way of using Google fonts via stylesheets (wouldn't know if/why you would want to change that?).

**Review by mjvkeulen — APPROVED** [review]

**Pull request merged**